sagenb/data/sage/html/history.html

```diff
@@ -3,13 +3,13 @@
 {% block title %}{{ gettext('Sage: History for %(u)s', u=username) }}{% endblock %}
 
 {% block javascript %}
-<script type="text/javascript" src="/javascript/sage/js/ws_list.js"></script>
+<script type="text/javascript" src="{{request.url_root}}/javascript/sage/js/ws_list.js"></script>
 {% endblock %}
 
 {% block page_id %}history-page{% endblock %}
 
 {% block body %}
-    <pre>{{ text | escape }}</pre>
```

Comment:
I think that @migeruhito actually pointed out that you merged some, but not all, of his stuff in #324 and that is what happened.